From 4d7737571101506d3056463e3c867c66cf2f8020 Mon Sep 17 00:00:00 2001
From: David Abdurachmanov <davidlt@cern.ch>
Date: Sun, 3 Nov 2013 18:15:55 +0100
Subject: [PATCH] Use /lib64 or/and /usr/lib64 on AArch64

All the libraries are in /usr/lib64 directory, /lib64 is a symlink
to /usr/lib64. There are no libraries in /lib or /usr/lib on
Fedora for AArch64.

Signed-off-by: David Abdurachmanov <davidlt@cern.ch>

diff --git a/Configure b/Configure
index 9c803dc..97bb4d3 100755
--- a/Configure
+++ b/Configure
@@ -1230,7 +1230,7 @@ if (!$no_krb5)
 		$withargs{"krb5-dir"} = "/usr/heimdal"
 			if $withargs{"krb5-dir"} eq "";
 		$withargs{"krb5-lib"} = "-L".$withargs{"krb5-dir"}.
-			"/lib -lgssapi -lkrb5 -lcom_err"
+			"/lib64 -lgssapi -lkrb5 -lcom_err"
 			if $withargs{"krb5-lib"} eq "" && !$IsMK1MF;
 		$cflags="-DKRB5_HEIMDAL $cflags";
 		}
@@ -1239,7 +1239,7 @@ if (!$no_krb5)
 		$withargs{"krb5-dir"} = "/usr/kerberos"
 			if $withargs{"krb5-dir"} eq "";
 		$withargs{"krb5-lib"} = "-L".$withargs{"krb5-dir"}.
-			"/lib -lgssapi_krb5 -lkrb5 -lcom_err -lk5crypto"
+			"/lib64 -lgssapi_krb5 -lkrb5 -lcom_err -lk5crypto"
 			if $withargs{"krb5-lib"} eq "" && !$IsMK1MF;
 		$cflags="-DKRB5_MIT $cflags";
 		$withargs{"krb5-flavor"} =~ s/^[Mm][Ii][Tt][._-]*//;
@@ -1249,7 +1249,7 @@ if (!$no_krb5)
 			}
 		}
 	LRESOLV:
-	foreach $lpath ("/lib", "/usr/lib")
+	foreach $lpath ("/lib64", "/usr/lib64")
 		{
 		foreach $lext ("a", "so")
 			{
-- 
1.8.0

